lib_expense/src/test/java/com/example/expenselogger/lib_expense/domain/fakes/FakeExpenseRepository.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.expenselogger.lib_expense.domain.fakes
+
+import com.example.expenselogger.cache.entity.ExpenseEntity
+import com.example.expenselogger.cache.repository.ExpenseRepository
+
+public class FakeExpenseRepository : ExpenseRepository {
```

Comment:
Why are you using a fake for the test

Reply:
I wanted to have a sort of 'implementation' to verify the behaviour.

Reply:
But you don't know the behavior, you're depending on an abstraction. If you change the implementation details in the cache module, this test will still pass because you're not coupled to the details of the cache module. You are only communicating via an interface

Reply:
Please will you available for a call maybe tomorrow or anytime you are free? This is not yet clear, and maybe a few other things